nss-day-cohort-22/Class-Website
# Updating json

## Link to Ticket

#3 

## Description of Proposed Changes
- Updated Cohort JSON with info from the spreadsheet
- Changed the name Angular.js to AngularJS
- Changed MySQL to SQL Server
- added `-webkit-justify-content: space-evenly` to the Contact Icons. I noticed when I pulled up the class website in Safari on my iPhone, the icons weren't spaced properly and researching said to add this. I wasn't able to fully check if it works because I couldn't figure out the Dev Inspection Tools on Safari but adding the webkit doesn't mess up any of the other formatting so I figured it didn't hurt to try it.

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout kn-updatingJson

```
Make sure all changes make sense and check for typos

Note: Courtney still doesn't have a photo, she's waiting to get it from the photographer. I reached out to see if there is another photo she wants us to use in the meantime 

## Impacted Areas in Application

List general components of the application that this PR will affect:

*  cohort.json
* tech.json


## Mentions @username

Tag users that need to review this code

@Chewieez @john-dulaney
